www/book.css

```diff
@@ -31,9 +31,17 @@ header, h1, strong {
 }
 }
 h1 { font-size: 105%; margin: 1.5em 0 0; }
+
+/* Auto-number section headings, but not on the main page. */
```

Comment:
Idea: how about we include the section numbers on the homepage and implement it with counter-reset?

Reply:
I'm a bit confused. When you say "section numbers", do you mean:

- Auto-number the parts?
- Auto-number the chapters?
- Auto-number the sections in each chapter?

Reply:
I  mean sections within chapters

Reply:
Right now the homepage doesn't show the sections. You want to add them? I guess that would look more like the printed book, not a bad idea. But perhaps a separate PR?

Reply:
ok